_posts/2020-09-28-Kamen-Kanev-cv.md

```diff
@@ -0,0 +1,127 @@
+
+### Technologies
+
+Ruby on Rails, React, GraphQL, PostgreSQL, Python, Django, NodeJS, RSpec, AWS, Capistrano
+
+## Professional Experience
+
+### Founder / CEO
+
+`April 2020 - September 2020`
+__[Bakalin BG](https://bakalin.bg/){:target="_blank" rel="noopener"}__
+
+We at Bakalin BG have the ambition of making the local specialty stores competitive to the supermarket chains.
+I and my partners have built the service from scratch, and I've learned a lot in various areas:
+Product vision, UX, Leadership, Logistics, Legal, Negotiation.
+One of the most noticeable achievements of mine is innovating with a delivery strategy
+that has improved our profit predictions with up to 20% on a yearly base.
+- Managing a team of 5 highly skilled young people
+- Contributing for 20-30% of the codebase: 2x React SPAs + Django GraphQL API
+- Developing a quantity management system(for grocery products) having up to 98% efficiency
+- Implementing agile practices in a **fully remote team** with a flexible work schedule
+- Making over 30 user interviews for validating product hypotheses
+
+### Team Lead
+
+`June 2019 - March 2020`
+__[Rifiniti, an FM:Systems Company](https://fmsystems.com/acq/rifiniti-is-now-fm-systems/){:target="_blank" rel="noopener"}__
+
+Working exclusively on Rails and NodeJS backed services. A Major part of my work included solving complex database problems
+coming from the requirements for fast analysis(<0.5s) of time-series data(over 500 million rows) for custom time ranges.
+Part of my contribution is:
+- Leading a team of 3 talented software engineers
+- Optimizing the load time of the main page with more than 30%
+- Speeding up the data deploys more than 2x
+
+### Senior Full-Stack Web Developer
+
+`May 2017 - May 2019`
+__[ReceiptBank](https://www.receipt-bank.com/){:target="_blank" rel="noopener"}__
+
+Worked alongside the best Rubists in town, maintaining a complex platform over Rails/GraphQL/Backbone/React tech stack.
+A key aspect of my job was balancing between what's desired from the clients' point of view and what's reasonable from the technological side.
+It was a process of following the best agile practices, keeping short feedback loops, and writing durable code -
+all that while working fully **remote**. My achievements were:
+- Being part of an agile cross-functional team and delivering advanced features for top tear clients
+- Completing multiple refactorings of business-critical UI components
+- Being part of a GDPR deletion functionality in a system with over 80 database tables
+- Doing zero-downtime migrations over tenths of millions of rows
+- Improving the code's test coverage with over 10%
+
+### Full-Stack Web Developer
+
+`June 2016 - May 2017`
+__[Tradeo](https://tradeo.com/){:target="_blank" rel="noopener"}__
+
+Stepping in a more responsible position, I started managing people and shifted focus towards the
```

Comment:
Merge both I think positions I think. Stepping here should not be used as this is not a novel

Reply:
I like them separated since makes my work experience looks greater. I'd hear advice about the wording at the beginning of the sentence.